pkg/utils/landscaper/blueprint.go

```diff
@@ -177,63 +230,66 @@ func (r *BlueprintRenderer) renderDeployItems(input *ResolvedInstallation, impor
 	// includes resolving target import references to target object references
 	deployItemTemplates := make(core.DeployItemTemplateList, len(executions))
 	for i, elem := range executions {
-		target := &core.ObjectReference{
```

Comment:
This change looks huge, but I actually just enclosed the coding which is responsible for resolving targets in an `if`, so that it is only executed if a target reference is given (this scenario caused a nil pointer error before).